neo4j/datadog_checks/neo4j/__init__.py

```diff
@@ -1,3 +1,8 @@
+import sys
+
+if not sys.executable:
+    sys.executable = None
+
```

Comment:
What is this for?

Reply:
removed. Not sure why I had needed it before.